include/seqan3/io/structure_file/format_vienna.hpp

```diff
@@ -28,6 +28,7 @@
 #include <range/v3/view/remove_if.hpp>
 #include <range/v3/view/transform.hpp>
 
+#include <seqan3/alphabet/structure/wuss.hpp>
```

Comment:
why this change?

Reply:
It's used in the file. I have no clue why it wasn't a problem before.